src/pages/workspaces/workspace/Analyses.js

```diff
@@ -11,9 +11,7 @@ import { ButtonOutline, Clickable, HeaderRenderer, Link, PageBox, spinnerOverlay
 import Dropzone from 'src/components/Dropzone'
 import { icon } from 'src/components/icons'
 import { DelayedSearchInput } from 'src/components/input'
-import {
-  AnalysisDeleter, AnalysisDuplicator, findPotentialNotebookLockers, getDisplayName, getFileName, getTool, notebookLockHash, stripExtension, tools
-} from 'src/components/notebook-utils'
+import { AnalysisDeleter, AnalysisDuplicator, findPotentialNotebookLockers, getDisplayName, getFileName, getTool, getToolFromRuntime, notebookLockHash, stripExtension, tools } from 'src/components/notebook-utils'
```

Comment:
As per the reformatter:

```suggestion
import {
  AnalysisDeleter, AnalysisDuplicator, findPotentialNotebookLockers, getDisplayName, getFileName, getTool, getToolFromRuntime, notebookLockHash,
  stripExtension, tools
} from 'src/components/notebook-utils'
```